CITATION.cff

```diff
@@ -0,0 +1,73 @@
+cff-version: 1.2.0
+message: If you use this software, please cite it as below.
+title: PyTorch
+authors:
+  - family-names: PyTorch Team
```

Comment:
The `authors` key is required and thus I've used the same string as we do in our package metadata:

https://github.com/pytorch/pytorch/blob/a75edfa97c9d985a337fbea7b9c0f4061153aaf0/setup.py#L1185